src/development/LocalTest/appsettings.json

```
@@ -39,11 +39,11 @@
   },
   "AllowedHosts": "*",
   "LocalPlatformSettings": {
-    "LocalAppMode": "file",
+    "LocalAppMode": "http",
     "LocalAppUrl": "http://localhost:5005",
     "LocalTestingStorageBasePath": "c:/AltinnPlatformLocal/",
     "AppRepositoryBasePath": "<Path to your repos folder where you have cloned the Altinn Studio application repositories>",
-    "LocalTestingStaticTestDataPath": "<Path to altinn-studio repo>/src/development/TestData/"
+    "LocalTestingStaticTestDataPath": "c:/Users/xivne/dev/altinn-studio//src/development/TestData/"
```

Revert hardcoded local path
```suggestion
    "LocalTestingStaticTestDataPath": "<Path to altinn-studio repo>/src/development/TestData/"
```